### PR TITLE
Update es.m3u

### PR DESCRIPTION
**Pull request by trincowski**
- Added new channels
- "TV3Cat.es" has 3 different links (cat/es/int), since depending on the VPN server, some might work while others won't

### CODE REVIEW

File: streams/es.m3u [hunk 1]
```diff
@@ -219,12 +247,26 @@ https://streamer1.streamhost.org/salive/logosH/master.m3u8
...
+#EXTINF:-1 tvg-id="OndaAlgeciras.es" status="online",Onda Algeciras TV (576p)
+https://cloudtv.provideo.es/live/algecirastv-livestream.m3u8
+#EXTINF:-1 tvg-id="OndaCadiz.es" status="online",Onda Cádiz (1080p)
+https://adc-hls.flumotion.com/playlist.m3u8
+#EXTINF:-1 tvg-id="OndaCadiz.es" status="online",Onda Cádiz 24h (720p)
+https://ondacadiztv.es:30443/octv/24h/playlist.m3u8
+#EXTINF:-1 tvg-id="OndaMadrid.es" status="online",Onda Madrid (720p)
+https://telemadridhls2-live-hls.secure2.footprint.net/egress/chandler/telemadrid/tvradio_1/index.m3u8
+#EXTINF:-1 tvg-id="OndaMadrid.es" status="online",Onda Madrid (360p)
+https://telemadridhls2-live-hls.secure2.footprint.net/egress/chandler/telemadrid/ondamadrid_1/index.m3u8
```
Comment (Alstruit) [inline streams/es.m3u:269]:
```suggestion
```
Better Adaptive Stream in previous one.

File: streams/es.m3u [hunk 2]
```diff
@@ -87,14 +93,22 @@ http://217.125.136.93:8080/canalsierradecadiz576.m3u8
 https://cdnlive.codev8.net/rtvalive/smil:channel1.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="CanalSurAndalucia.es" status="timeout",Canal Sur Andalucía (720p)
 https://cdnlive.shooowit.net/rtvalive/smil:channel1.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="CanalTaronja.es" status="online",Canal Taronja Osona i Moianés (1080p) [Not 24/7]
+https://ingest1-video.streaming-pro.com/videostream/ctvic/playlist.m3u8
 #EXTINF:-1 tvg-id="CanalTaronja.es" status="blocked",Canal Taronja Osona i Moianés (480p) [Not 24/7]
 https://taronjavic.streaming-pro.com/hls/vic.m3u8
+#EXTINF:-1 tvg-id="CartoonNetworkSpain.es" status="online",Cartoon Network Spain (720p)
+https://playout.cdn.cartoonnetwork.com.br/playout_02/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/es.m3u:101]: [already in br.m3u](https://github.com/iptv-org/iptv/blob/639e9c5a9d75ac84099c0e04c3ad9b6f978b94e1/streams/br.m3u#L49-L50)

File: streams/es.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply streams/es.m3u:267]: Can still serve as a backup link.

File: streams/es.m3u [hunk 3]
```diff
@@ -399,3 +457,27 @@ https://streaming007.gestec-video.com/hls/artequatreTVA.m3u8
 https://cloud3.streaminglivehd.com:1936/universfaller/universfaller/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="timeout",Urban Revolution
 https://www.urbanrevolution.es:444/live/5e6d8470a3832/index.m3u8
+#EXTINF:-1 tvg-id="" status="online",7 TeleValencia (576p)
+https://live-edge-ams-1.cdn.enetres.net/9E9557EFCEBB43A89CEC8FBD3C500247022/live-500/index.m3u8
```
Comment (LaneSh4d0w) [inline streams/es.m3u:461]: 404. Not 24/7 ?

Comment (trincowski) [reply streams/es.m3u:461]: It's possible, it was working around 10h ago.

Comment (trincowski) [reply streams/es.m3u:461]: > 404. Not 24/7 ?

Actually, it is working right now:

7TeleValencia

**Review by Alstruit — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**